### PR TITLE
Upgrade harvester chart and bump kubevirt and cdi

### PR DESCRIPTION
**Pull request by guangbochen**
https://github.com/rancher/harvester/issues/262, https://github.com/rancher/harvester/issues/267

**Description:**
1. bump  kubevirt to v0.35.0 and CDI to v1.26.1 respectively
2. upgrade `controller-gen` to v0.4.1 to make it compatible with k8s 1.19

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,55 +6,63 @@ replace (
...
-	k8s.io/client-go => k8s.io/client-go v0.18.0
+	k8s.io/api => k8s.io/api v0.18.6
+	k8s.io/client-go => k8s.io/client-go v0.18.6
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.16.4
 	k8s.io/code-generator => k8s.io/code-generator v0.18.3
-	kubevirt.io/client-go => github.com/cnrancher/kubevirt-client-go v0.31.1-0.20200715061104-844cb60487e4
-	kubevirt.io/containerized-data-importer => github.com/cnrancher/kubevirt-containerized-data-importer v1.22.0-apis-only
+	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20191107075043-30be4d16710a
+	kubevirt.io/client-go => github.com/cnrancher/kubevirt-client-go v0.20.2-0.20201124032527-ff3943be35bf
+	kubevirt.io/containerized-data-importer => github.com/cnrancher/kubevirt-containerized-data-importer v1.22.1-0.20201127032458-4f73ed532d31
```
Comment (gitlawr) [inline go.mod:23]: Why is this v1.22.1 when it requires kubevirt.io/containerized-data-importer v1.25.0?

Comment (guangbochen) [reply go.mod:23]: the tag is auto-generated by the go mod vendor however both the code and commit are correct.

File: go.mod [hunk 2]
```diff
@@ -6,55 +6,63 @@ replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v12.2.0+incompatible
 	github.com/crewjam/saml => github.com/rancher/saml v0.0.0-20180713225824-ce1532152fde
 	github.com/dgrijalva/jwt-go => github.com/dgrijalva/jwt-go v3.2.1-0.20200107013213-dc14462fd587+incompatible
+	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.3.1
 	github.com/knative/pkg => github.com/rancher/pkg v0.0.0-20190514055449-b30ab9de040e
+	github.com/openshift/api => github.com/openshift/api v0.0.0-20191219222812-2987a591a72c
+	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20191125132246-f6563a70e19a
+	github.com/operator-framework/operator-lifecycle-manager => github.com/operator-framework/operator-lifecycle-manager v0.0.0-20190128024246-5eb7ae5bdb7a
```
Comment (gitlawr) [inline go.mod:13]: These replacements look like unnecessary

Comment (guangbochen) [reply go.mod:13]: those are used by the CDI importer

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**